### PR TITLE
Consolidate duplicate pkg/controller/utils imports

### PR DESCRIPTION
**Pull request by 2uasimojo**
There were a few files where pkg/controller/utils was being imported twice with different aliases. Consolidate.

Note that in these cases I chose to retain the `controllerutils` alias, since `utils` is so generic. However, I didn't go back through and change all the other files where that alias isn't used.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/hold

I put this on top of #441 to avoid a messier rebase, and wanting to merge that one first since it's more important. Rebase this once #441 merges. Meanwhile, [review just this change](https://github.com/openshift/aws-account-operator/pull/443/commits/0f0153606442add5d79ab8449b3eb0309fd2db45).

**Review by clcollins — APPROVED** [review]
/lgtm

@2uasimojo looks like #441 has merged.  I think the HOLD can be removed?

**Comment by 2uasimojo** [comment]
Thanks @clcollins. Rebased.

/hold cancel

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**
